File: recaudacion/2.Codigo/frontend/src/app/core/models/cliente.ts
export class Cliente {
    clienteId: number;
    tipoDocumentoIdentidadId: number;
    tipoDocumentoIdentidad:any;
    TipoDocumentoNombre:string;
    numeroDocumento: string;
    nombre: string;
    direccion: string;
    correo: string;
    estado: boolean;
    usuarioCreador: string;
    usuarioModificador: string;
    index:number;
}

export class ClienteFilter {
    pageNumber: number;
    pageSize: number;
    sortColumn:string;
    sortOrder: string;
    tipoDocumentoIdentidadId: number;
    numeroDocumento: string;
    nombre:string;
    estado: boolean;
}

File: recaudacion/2.Codigo/backend/RecaudacionApiLiquidacion/Domain/Cliente.cs
namespace RecaudacionApiLiquidacion.Domain
{
    public class Cliente
    {
        public int ClienteId { get; set; }
        public int TipoDocumentoIdentidadId { get; set; }
        public string TipoDocumentoNombre { get; set; }
        public string NumeroDocumento { get; set; }
        public string Nombre { get; set; }
        public string Direccion { get; set; }
        public string Correo { get; set; }
        public bool Estado { get; set; }
    }
}
